### PR TITLE
Feat/add unsupported archive error handling

### PR DESCRIPTION
**Pull request by agatakrajewska**
- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Adding an extra error handling, when no layers found in image provided.

### CODE REVIEW

File: lib/extractor/docker-archive/layer.ts [hunk 1]
```diff
@@ -83,6 +83,10 @@ function getLayersContentAndArchiveManifest(
     .map((layerName) => layers[layerName])
     .reverse();
 
+  if (filteredLayers.length === 0) {
+    throw new Error("We found no layers in the provided image");
+  }
```
Comment (ivanstanev) [inline lib/extractor/docker-archive/layer.ts:88]: Do we have a test case for this, any way we can add coverage?

Comment (agatakrajewska) [reply lib/extractor/docker-archive/layer.ts:88]: Yeees, was just thinking that! 👍

Comment (agatakrajewska) [reply lib/extractor/docker-archive/layer.ts:88]: Done!

**Comment by agatakrajewska** [comment]
> Great job! 🎉
> I was wondering if the commit type should be `fix` instead of `feat`, what do you think?

Yes, so the first commit is, but the second one, I think it should be `feat` as it changes the functionality of the code, I think! 🤔

**Pull request merged**